guardian/grid
# Add logging contexts to cropper

## What does this change?

Use the RequestLoggingFilter introduced in #3719 to get stable request IDs on cropper endpoints, and passes logMarkers through into functions defined in and used by cropper. It should be easier now to relate error logs in cropper to other logs created by the same request.

## How should a reviewer test this change?

try to create a full-frame crop of https://media.test.dev-gutools.co.uk/images/a36e8475249d99bf45fce2f2288aef8ca8b91a18 , which should (currently) fail. Find an error log caused by this, then create a new search on the `requestId` marker - you should now see all related logs from that request.

## How can success be measured?

## Who should look at this?


## Tested? Documented?
